tailwind.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     './public/*.html',
   ],
   mode: 'jit',
+  darkMode: 'selector',
```

Comment:
allows manual toggle (not just based on system settings) to read the `dark:{class}` tags
https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually